### PR TITLE
Docs: Clean up of 'fetch' module docs

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This is part of a series of module doc cleanups.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
fetch

##### ANSIBLE VERSION
v2.8

### CODE REVIEW

File: lib/ansible/modules/files/fetch.py [hunk 1]
```diff
@@ -1,101 +1,100 @@
...
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['stableinterface'],
                     'supported_by': 'core'}
 
-
 DOCUMENTATION = '''
 ---
 module: fetch
-short_description: Fetches a file from remote nodes
+short_description: Fetche files from remote nodes
```
Comment (samccann) [inline lib/ansible/modules/files/fetch.py]: Fetch files...  (remove the extra e)

File: lib/ansible/modules/files/fetch.py [hunk 2]
```diff
@@ -1,101 +1,100 @@
...
-    - This module is also supported for Windows targets.
+- When running fetch with C(become), the M(slurp) module will also be
+  used to fetch the contents of the file for determining the remote
+  checksum. This effectively doubles the transfer size, and
+  depending on the file size can consume all available memory on the
+  remote or local hosts causing a C(MemoryError). Due to this it is
+  advisable to run this module without C(become) whenever possible.
+- Prior to Ansible-2.5 this module would not fail if reading the remote
+  file was impossible unless C(fail_on_missing) was set.
+- In Ansible-2.5+, playbook authors are encouraged to use C(fail_when)
```
Comment (samccann) [inline lib/ansible/modules/files/fetch.py]: In Ansible 2.5 or later,...

**Review by samccann — DISMISSED** [review]
Thanks for the fixes... just a few minor things to change.

File: lib/ansible/modules/files/fetch.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/files/fetch.py]: Good catch ! Fixed.

File: lib/ansible/modules/files/fetch.py (hunk 2 above)
Comment (dagwieers) [reply lib/ansible/modules/files/fetch.py]: Right, thanks !

**Comment by dagwieers** [comment]
Please re-review !

**Pull request merged**